spec/features/accessibility/idv_pages_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'rails_helper'
 require 'axe/rspec'
 
-feature 'Accessibility on IDV pages', :js do
+feature 'Accessibility on IDV pages', :js, :accessibility do
```

Comment:
What did you try? Based on: https://www.relishapp.com/rspec/rspec-core/v/3-5/docs/configuration/exclude-pattern

I would expect this to work:

 `--exclude-pattern "**/features/accessibility/*_spec.rb"`